gradle.properties

```diff
@@ -1,11 +1,9 @@
-projectVersion=3.0.2-SNAPSHOT
+projectVersion=3.1.0-SNAPSHOT
 projectGroup=io.micronaut.xml
-
 micronautDocsVersion=2.0.0
-micronautVersion=3.0.3
-micronautTestVersion=3.0.5
-
-groovyVersion=3.0.9
+micronautVersion=3.4.3
```

Comment:
Note to self: will have to streamline things so that we don't have to duplicate versions in `gradle.properties`

Reply:
I'm thinking maybe these five properties:

1) micronautDocsVersion
2) micronautVersion
3) micronautTestVersion
4) groovyVersion
5) spockVersion

should also move to `libs.versions.toml` file? Or it should be part of Micronaut internal Gradle plugins?